spec/controllers/organizations_controller_spec.rb

```diff
@@ -5,6 +5,7 @@
 
   let(:organization)  { GitHubFactory.create_owner_classroom_org }
   let(:user)          { organization.users.first                 }
+  let(:student)       { GitHubFactory.create_classroom_student   }
```

Comment:
@johndbritton, that comment from me is outdated. This `student` user is only used in the context `user without admin privilege on the organization` now. So I think `student` should be enough.